<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>777</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>777</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>777</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DAIHATSU DIESEL MFG.CO.,LTD</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>C:/Users/USER/Downloads/image_2024_10_09T06_33_11_459Z.png</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="text_browsefolder">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>78</y>
      <width>721</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border-radius: 10px; /* Bo tròn góc với bán kính 10px */
    border: 1px solid #ccc; /* Đường viền màu xám nhạt */
    background-color: white; /* Màu nền */
    padding: 5px; /* Khoảng cách bên trong */</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delete">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>128</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #7EC0EE;
    color: black;
    border-radius: 10px;
    border: 3px solid #FFFFFF;</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_graphitems">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>138</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Graph Items</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list_process">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>490</y>
      <width>871</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border-radius: 10px; /* Bo tròn góc với bán kính 10px */
    border: 1px solid #ccc; /* Đường viền màu xám nhạt */
    background-color: white; /* Màu nền */
    padding: 5px; /* Khoảng cách bên trong */</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>638</y>
      <width>871</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
    border: 1px solid #ccc;  /* Đường viền */
    border-radius: 10px;  /* Bo tròn góc */
    text-align: center;  /* Hiển thị phần trăm bên ngoài */
    padding-right: 15px;  /* Tạo khoảng trống để phần trăm hiện bên ngoài */
    background-color: #f3f3f3;  /* Màu nền thanh trống */
}

QProgressBar::chunk {
    background-color: #7EC0EE;  /* Màu xanh của thanh tiến trình */
    border-radius: 8px;  /* Bo góc cho thanh tiến trình */
}</string>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_start">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>670</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color:#e67e22;
    color:black;
    border-radius: 10px;
    border: 5px solid #FFFFFF;</string>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_language">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>700</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
    border: 1px solid #ccc; /* Đường viền màu xám nhạt */
    background-color: white; /* Màu nền */
    padding: 5px; /* Khoảng cách bên trong */</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/USER/Downloads/img_daihatsu_logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>130</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #7EC0EE;
    color: black;
    border-radius: 10px;
    border: 3px solid #FFFFFF;</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_graphitems_2">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>40</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/USER/Downloads/search.svg</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_browsefolder">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #7EC0EE;
    color: black;
    border-radius: 10px;
    border: 3px solid #FFFFFF;</string>
    </property>
    <property name="text">
     <string>Browse Folder</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_graphitems">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
    border: 1px solid #ccc; /* Đường viền màu xám nhạt */
    background-color: white; /* Màu nền */
    padding: 5px; /* Khoảng cách bên trong */</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_graphitems_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>700</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/USER/Downloads/globe.svg</pixmap>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_data">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>871</width>
      <height>251</height>
     </rect>
    </property>
    <property name="gridStyle">
     <enum>Qt::DotLine</enum>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <column>
     <property name="text">
      <string>NAME</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>UPPER LIMIT</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>LOWER LIMIT</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>EXPAN NUMBER</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
